Add node_isolation_heartbeat_timeout tunable property (#1186)
redpanda-data/documentation: modules/get-started/pages/release-notes/operator.adoc

== Redpanda Operator v25.1.x

> link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.1.x/operator/CHANGELOG.md[Changelog^].

Reviewer: ```suggestion
link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.1.x/operator/CHANGELOG.md[Changelog^]
```

=== Redpanda Console v3

> This version deploys Redpanda Console v3, which includes unified authentication and authorization between the Console and Redpanda, including user impersonation. For more information, see xref:console:config/security/authentication.adoc[].

Reviewer 2: Oh dear, this is actually not true 😓 We've had to push off making that change due to complications in getting the CRD updated.

The v25.1.1 _chart_ will deploy console v3 but v25.1 and v25.2 of the operator is still on console v2.

Reviewer 3: @chrisseto So Console v3 is supported if you use the Redpanda Helm chart, but not if you use the operator?

Reviewer 2: Correct!

> The Redpanda Helm chart now deploys **Redpanda Console v3** by default. Console v3 introduces support for unified authentication and authorization with Redpanda, including user impersonation.

Reviewer 3: ```suggestion
To try Redpanda Console v3 in Kubernetes, you can deploy Redpanda using the Redpanda Helm chart instead of the Redpanda Operator. Redpanda Console v3 introduces support for unified authentication and authorization with Redpanda, including user impersonation.
```

See also: xref:upgrade:k-compatibility.adoc[].

=== Unified Helm installation of Redpanda Operator and CRDs

Starting in Redpanda Operator v25.1.1, you can install both the Redpanda Operator and its required custom resource definitions (CRDs) using a single Helm command:

[,bash,subs="attributes+"]
----
helm repo add redpanda https://charts.redpanda.com
helm repo update
helm upgrade --install redpanda-controller redpanda/operator \
  --namespace <namespace> \
  --create-namespace \
  --version {latest-operator-version} \
  --set crds.enabled=true
----

> === Kafka Connect integration removed from Helm chart

Reviewer 2: I'm not sure if we should document this but the `connectors` field on the CRD is still present (Again, complications in updating the CRD :( ) it will just be ignored if it's specified.

I'll update the docs configuration to hide the connectors field to help minimize confusion.

Reviewer 3: Thanks, Chris. If it's hidden, it shouldn't get pulled into our CRD reference docs.

Starting with version 25.1.x of the Redpanda Helm chart, Kafka Connect is no longer bundled as a subchart.

If you require Kafka Connect:

- You must install the `connectors` Helm chart separately after deploying Redpanda.
- This change affects both Helm and Operator-based Redpanda deployments on Kubernetes.

> link:<link-to-knowledge-base-article>[See the migration guide^] for instructions.

Reviewer: same comments about list as in helm-charts.adoc, especially important to fix link

> To check if you're currently using Redpanda's managed Kafka Connect:

Reviewer: ```suggestion
To check if you're currently using Redpanda's managed Kafka Connect, run:
```
